app/common/model/src/main/java/io/syndesis/common/model/ModelData.java

```diff
@@ -73,7 +72,7 @@ public void setDataFromJson(JsonNode json) throws JsonProcessingException {
     public T getData() throws IOException {
         if (data == null && kind != null && json != null) {
             @SuppressWarnings("unchecked")
```

Comment:
maybe we can also get rid of the `@SuppressWarning` now

Reply:
True. I'm working on the next round already, adding that :)